source/retryable-writes/retryable-writes.rst

```diff
@@ -208,16 +214,24 @@ execute the command again. Consider the following pseudo-code:
   }
 
 When selecting a writable server for the first attempt of a retryable write
-operation, drivers MUST raise a client-side error if the server’s maximum wire
+command, drivers MUST raise a client-side error if the server's maximum wire
 version does not support retryable writes. If the server selected for a retry
 attempt does not support retryable writes (e.g. mixed-version cluster), retrying
-is not possible and drivers MUST raise the original network error to the user.
+is not possible and drivers MUST raise the original error to the user.
 
 When retrying a write command, drivers MUST resend the command with the same
 transaction ID. Drivers MAY resend the original wire protocol message (see:
 `Can drivers resend the same wire protocol message on retry attempts?`_). If the
-second attempt also fails with a network error, drivers MUST raise its
-corresponding error to the user.
+second attempt also fails with retryable error, drivers MUST update their
+topology according to the SDAM spec (see: `Error Handling`_) and raise that
+error to the user. Drivers MUST NOT attempt to retry a write command with the
+same transaction ID more than once.
+
+In the case of a multi-statement write operation split across multiple write
+commands, a failed retry attempt will also interrupt execution of any additional
+write operations in the batch (regardless of the ordered option). This is no
+different than if a retryable error had been encountered without retryable
+behavior enabled or supported by the driver.
```

Comment:
This relates to my comment in https://github.com/mongodb/specifications/pull/219#discussion_r145488556.